h2/src/main/org/h2/command/dml/Select.java

```diff
@@ -114,6 +114,9 @@
 
     private int[] groupByCopies;
 
+    /**
+     * Whether this query contains aggregate functions.
```

Comment:
This flag is set when `SELECT` statement contains (non-window) aggregate functions, `GROUP BY` clause, or `HAVING` clause. `GROUP BY` and `HAVING` may be used even without aggregate functions.

Documentation for `setGroupQuery()` is also incomplete.